src/nemos/utils.py

```diff
@@ -252,37 +331,30 @@ def nan_pad_conv(
         )
 
     start, end = adjust_indices[filter_type]
-
+    # convert to jax ndarray
+    conv_trials = jax.tree_map(jnp.asarray, conv_trials)
     try:
-        conv_trials = jnp.asarray(conv_trials)
         if conv_trials.ndim != 4:
-            raise ValueError(
-                "conv_trials must be an iterable of 3D arrays or a 4D array!"
+            raise AttributeError(
```

Comment:
```suggestion
            raise ValueError(
```
Shouldn't this be a ValueError, to be consistent with the docstring, and the error with identical error message raised below?